hub/powertoys/color-picker.md

```diff
@@ -48,28 +48,92 @@ The copied color will be stored in your clipboard in the format that is configur
 
 The editor lets you see the history of picked colors (up to 20) and copy their representation in any predefined string format. You can configure which color formats are visible in the editor and in what order that they appear. This configuration can be found in PowerToys Settings.
 
-The editor also allows you to fine-tune any picked color or get a new similar color. Editor suggests two lighter and two darker shades of the currently selected color. Clicking on any of those alternative color shades will add the selection to the history of picked colors. It will appear at the top of the colors history list.
+The editor also allows you to fine-tune any picked color or get a new similar color. Editor suggests two lighter and two darker shades of the currently selected color. Selecting any of those alternative color shades will add the selection to the history of picked colors. It will appear at the top of the colors history list.
 
-The color in the middle represents your currently selected color from the colors history. By clicking on it, the fine-tuning configuration control will appear, where you can change HSV or RGB values of the current color. Pressing <kbd>Select</kbd> will add newly configured color into the colors history.
+The color in the middle represents your currently selected color from the colors history. By clicking on it, the fine-tuning configuration control will appear, where you can change HSV or RGB values of the current color. Using the <kbd>Select</kbd> button will add the newly configured color into the colors history.
 
 ![ColorPicker Editor window.](../images/pt-colorpicker-editor.gif)
 
-To remove any color from the colors history, right click a color and select **Remove**.
+To remove any color from the colors history, right-click a color and select **Remove**.
+
+To export the colors history, right-click a color and select **Export**. The exported values can be grouped by colors or formats.
 
 ## Settings
 
 Color Picker has following settings:
 
 - Activation shortcut
 - Behavior of activation shortcut
-- Format of a copied color (HEX, RGB, etc.)
+- The default format of a copied color (HEX, RGB, etc.)
 - Show the name of the color. The shown color name will be a high-level representation of the color. Example: Light Green, Green, Dark Green.
 - Order and presence of color formats in the editor
 
-![ColorPicker Settings screenshot.](../images/pt-colorpicker-settings.png)
+![ColorPicker Settings screenshot.](../images/pt-colorpicker-settings.gif)
+
+To reorder the colors, select the **...** buttom on the right and select **Move up** or **Move down**.
```

Comment:
What is a _buttum_ ?

Reply:
good catch, https://github.com/MicrosoftDocs/windows-dev-docs/pull/4198 has fix